avoid leading spaces causing unbalanced quotes to be added:

$ ./spam-unparseable-mail.php
from[  "blah" <blah@example.com>]
address["  "blah" <blah@example.com>]

--- eventum/include/class.mail.php~	2006-07-17 23:53:46.000000000 +0300
+++ eventum/include/class.mail.php	2006-07-17 23:55:24.587626835 +0300
@@ -211,7 +211,7 @@
     {
         // check if we have a <
         if ((strstr($address, '<')) && (!Mime_Helper::isQuotedPrintable($address))) {
-            $address = stripslashes($address);
+            $address = stripslashes(trim($address));
             // is the address in the format 'name' <address> ?
             if ((strstr($address, "'")) || (strstr($address, "."))) {
                 $first_part = substr($address, 0, strpos($address, '<') - 1);
